com.unity.render-pipelines.universal/ShaderLibrary/GlobalIllumination.hlsl

```diff
@@ -9,11 +9,11 @@
 // If lightmap is not defined than we evaluate GI (ambient + probes) from SH
 // We might do it fully or partially in vertex to save shader ALU
 #if !defined(LIGHTMAP_ON)
-// TODO: Controls things like these by exposing SHADER_QUALITY levels (low, medium, high)
-    #if defined(SHADER_API_GLES) || !defined(_NORMALMAP)
+    // TODO: LOW == defined(SHADER_API_GLES) || !defined(_NORMALMAP)
+    #if _SHADER_QUALITY_LOW
         // Evaluates SH fully in vertex
         #define EVALUATE_SH_VERTEX
-    #elif !SHADER_HINT_NICE_QUALITY
+    #elif _SHADER_QUALITY_MEDIUM
```

Comment:
I wonder if we should profile and check the impact of removing this as well.

Reply:
These are not active in this PR, but they're there if you really really want them.

Reply:
Removed.